dnf-behave-tests/features/plugins-core/needs-restarting-conf-files.feature

```diff
@@ -0,0 +1,122 @@
+Feature: Add package to needs-restarting using config files
+
+Background:
+    Given I enable plugin "needs_restarting"
+      And I use repository "dnf-ci-fedora"
+      And I move the clock backward to "before boot-up"
+      And I execute dnf with args "install lame kernel basesystem glibc wget abcde"
+      And I move the clock forward to "the present"
+      And I use repository "dnf-ci-fedora-updates"
+    Given I create directory "/etc/dnf/plugins/needs-restarting.d"
+      And I create and substitute file "/etc/dnf/plugins/needs-restarting.d/wget.conf" with
+      """
+      wget
+      """
+      Given I do not set config file
+
+    @bz1810123
+    Scenario: restarting needed after updating package in conf files
+        Given I execute dnf with args "upgrade wget"
+         When I execute dnf with args "needs-restarting -r"
+         Then the exit code is 1
```

Comment:
Unrelated to your PR, so consider this a rhetorical question :slightly_smiling_face: but is it really returning a non-zero exit status when there are packages to restart? That doesn't make sense...